README.md

```diff
@@ -5,10 +5,13 @@ This repo contains deployment scripts executed during project builds on [Travis
 The scripts should work across a number of repos that include a `Dockerfile`. They are intended to be included as a [Git submodule](https://git-scm.com/docs/git-submodule) within the code repo.
 
 Including this repo within your own (as detailed below) will:
-* Build and tag the Docker image as detailed in the repo's `Dockerfile`. This happens on PR, Tag and change to `master` branch.
-* Push the image to the docker repo
-* If the Travis build has been triggered by a PR a new stack will be created within the dev environment in Rancher. The URL to thatstack will be published in the PR's comments
-
+* Build and tag the Docker image as detailed in the repo's `Dockerfile`. This happens on PR creation and update, Tagging (including Release) and commits to the `master` branch.
+* Push the image to the appropriate docker repo
+* If the Travis build has been triggered by a PR a new stack will be created within the dev environment in Rancher. The rancher and docker templates used to build the the stack are pulled from the [nhsuk/nhsuk-rancher-templates](https://github.com/nhsuk/nhsuk-rancher-templates) repo on GitHub. The specific compose files are determined by the env var RANCHER_TEMPLATE_NAME and the subdirectory '0' is always used.
```

Comment:
Double 'the' at '...build the the stack...'

Maybe surround `RANCHER_TEMPLATE_NAME` with back ticks?